Framework/Geometry/test/ComponentInfoTest.h

```diff
@@ -704,6 +704,26 @@ class ComponentInfoTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(panel.topLeft, 3);
     TS_ASSERT_EQUALS(panel.topRight, 15);
   }
+
+  void test_new_demo_instrument() {
```

Comment:
Please delete this method, I was only using it for my own TDD, and it certainly doesn't belong here. I only committed it for your reference as a temporary.